lib/sepa_king/transaction.rb

```diff
@@ -4,6 +4,8 @@ class Transaction
     include ActiveModel::Validations
     extend Converter
 
+    DEFAULT_REQUESTED_DATE = Date.new(1999, 1, 1).freeze
```

Comment:
What about `NEXT_POSSIBLE_DATE` or `EARLIEST_POSSIBLE_DATE` to tell something about the meaning? It should be documented in the README, too.